Formula/macvim.rb

```diff
@@ -55,6 +54,7 @@ def install
                           "--with-macarchs=#{Hardware::CPU.arch}"
     system "make"
 
+    rm_rf Dir["src/MacVim/build/Release/MacVim.app/Contents/Frameworks/Sparkle.framework"]
```

Comment:
```suggestion
    (buildpath/"src/MacVim/build/Release/MacVim.app/Contents/Frameworks/Sparkle.framework").rmtree
```
Also, we should move this to the start of the `install` method.